vaadin-split-layout.html

```diff
@@ -0,0 +1,33 @@
+<link rel="import" href="color.html">
+
+<dom-module id="material-split-layout" theme-for="vaadin-split-layout">
+  <template>
+    <style>
+      [part="splitter"] {
+        min-width: 8px;
+        min-height: 8px;
+        background-color: var(--material-split-layouts-splitter-color, #000);
+        transition: 0.1s background-color;
+      }
+
+      [part="handle"] {
+        display: flex;
+        align-items: center;
+        justify-content: center;
+      }
+
+      [part="handle"]::after {
+        content: "";
+        display: block;
+        width: 2px;
+        height: 24px;
+        background-color: var(--material-split-layouts-handle-color, #fff);
```

Comment:
Use `--material-body-background-color` instead, let’s not introduce a new color just for this.